rte/mo_rte_lw.F90

```diff
@@ -205,12 +211,22 @@ function rte_lw(optical_props, top_at_1, &
         !$acc enter data copyin(optical_props%tau)
         error_msg =  optical_props%validate()
         if(len_trim(error_msg) > 0) return
-        call lw_solver_noscat_GaussQuad(ncol, nlay, ngpt, logical(top_at_1, wl), &
-                              n_quad_angs, gauss_Ds(1:n_quad_angs,n_quad_angs), gauss_wts(1:n_quad_angs,n_quad_angs), &
+        if (present(lw_Ds)) then
+          lw_Ds_wt = gauss_wts(1,1)
```

Comment:
Why make this assignment rather than simply passing the value?

Reply:
My notes say to make a clear variable for passing it in BUT I can pass in directly if that is better.

Reply:
Changed to pass the value in directly